docs/index.html

```diff
@@ -9,19 +9,22 @@
   <link rel="stylesheet" href="./css/tailwind.css">
   <style>
     h1 {
-        font-weight: bold;
-        font-size: 200%;
+      font-weight: bold;
```

Comment:
I'd drop these changes. Making existing code consistent is one thing, but if every PR tweaked the indent size, it would end up quite annoying.

Reply:
Would you be open to me adding linting to the CI to enforce a style? My personal preference is always to leave the formatting to the machines so I don't have to think about it :-).

Reply:
That sounds like a discussion for another issue or PR.

Anyway, it was more of a drive-by review, I'm not the one to decide on this. But in my (very limited) experience to the web dev tools (like bundlers), these tools tend to bring a lot more churn than asking people to revert a couple of lines every now and then.